dict/portuguese.yml

```diff
@@ -0,0 +1,291 @@
+    'BT': [Butão, Butanês*, Butanense*, Butani*, Butâni*, Timbu, Thimphu, Timpu, Thimbu]
+    'IN': [Índia, Indian*, Bombaim, Mumbai, Nova Deli, Nova Delhi, Nova Déli]
+    'IR': [Irã, Irão, Iranian*, Teerã, Teerão]
+    'LK': [Sri Lanka, Sri Lanca, Seri Lanca, Ceilão, Cingalês*, Srilankês*, Srilanquês*, Colombo, Columbo]
+    'MV': [Maldivas, Maldiv*, Maldívi*]
+    'NP': [Nepal, Nepalês, Nepales*, Catmandu]
+    'PK': [Paquistão, Paquistanês*, Islamabade, Islamabad]
+
+  SOUTH-EAST:
+    'BN': [Brunei, Brunean*, Bruneín*]
+    'ID': [Indonésia, Indonési*, Jacarta]
+    'KH': [Camboja, Cambodja, Cambojan*, Phnom Penh, Pnom Pen, Pnom Pene, Penome Pene]
+    'LA': [Laos, Laosian*, Laocian*, Vientiane, Vienciana]
+    'MM': [Myanmar, Birmânia, Mianmarense*, Birmanês*, Birmã*, Birman*, Birmanian*, Birmanense*, Rangum, Yangon, Nepiedó, Naypyidaw, Nay Pyi Taw]
+    'MY': [Malásia, Malaio*, Malaia*, Malási*, Kuala Lumpur, Cuala Lumpur, Quala Lumpur, Cualalampur, Putrajaia, Putrajaya]
+    'PH': [Filipinas, Filipin*,  Manila]
+    'SG': [Singapura, Cingapura, Singapurense*, Singapuran*]
+    'TH': [Tailândia, Tailandês*, Banguecoque, Bangkok, Bancoque]
+    'TL': [Timor-Leste, Timorense*, Maubere*, Díli]
+    'VN': [Vietnã, Vietname, Vietnamit*, Vietnamês*, Vietnamense*, Vietnamian*, Hanói, Ho Chi Minh, Saigon, Saigão]
+
+  WEST:
+    'AE': [Emirados Árabes Unidos, EAU, Emiradense*, Emirátic*, Dubai, Abu Dhabi, Abu Dabi]
+    'AM': [Arménia, Armênia, Arméni*, Armêni*, Erevã, Erevan, Ierevan, Ierevã]
+    'AZ': [Azerbaijão, Azeri*, Bacu, Baku]
+    'BH': [Bahrein, Barém, Barein, Bareine, Bareinit*, Baremit*, Baremense*, Baremês*, Manama]
+    'CY': [Chipre, Cipriot*, Cípri*, Ciprense*, Nicósia]
+    'GE': [Geórgia, Georgian*, Geórgic*, Tiblíssi, Tbilisi]
+    'IL': [Israel, Israelit*, Israelense*, Jerusalém]
+    'IQ': [Iraque, Iraquian*, Bagdá, Bagdade]
+    'JO': [Jordânia, Jordanian*, Jordan*, Amã]
+    'KW': [Kuwait, Kuwaitian*, Koweitian*, Cuwaitian*, Coveitian*, Kowaitian*, Cuvait*, Kuwait*, Cuait*, Koweit, Coveite, Couaite, Quaite]
+    'LB': [Líbano,  Libanês*, Beirute]
+    'OM': [Omã, Omão, Omanense*, Omanian*, Omani, Mascate]
+    'PS': [Palestina, Palestin*, Gaza, Cisjordânia]
+    'QA': [Catar, Qatar, Catarense*, Catari*, Catarian*, Doha, Doa]
+    'SA': [Arábia Saudita, Saudit*, Riade]
+    'SY': [Síria, Siríac*, Sírio*, Síria*, Damasco]
+    'TR': [Turquia, Turc*, Ancara, Istambul]
+    'YE': [Iémen, Iêmen, Iemenit*, Saná, Sanaa, Sanaá]
+
+EUROPE:
+  EAST:
+    'BG': [Bulgária, Búlgar*, Sófia]
+    'BY': [Bielorrússia, Belarus, Bielorruss*, Belaruss* Minsk, Minsque]
+    'CZ': [Chéquia, Tchéquia, República Checa, República Tcheca, Chec*, Praga]
+    'HU': [Hungria, Húngar*, Budapeste]
+    'MD': [Moldávia, Moldova, Moldávi*, Moldávic*, Moldav*, Moldov*, Quixinau, Quichinau, Chisinau, Quixineve]
```

Comment:
Móldavi* should match Móldavia & Moldov* should match Moldova

Reply:
Changed